chips/apollo3/src/iom.rs

```diff
@@ -505,6 +505,10 @@ impl<'a> Iom<'_> {
                 data_popped = data_idx + 4;
             }
 
+            for _i in 0..3000 {
```

Comment:
The comment on lines 483-488 do provide some background for this, but looking at the errata, the number 3000 seems arbitrary. I also can't find a specific errata that points to this problem. Can you help point in to where in the errata PDF? 

Since 3000 nops twice ends up being pretty significant in time: (3000 * 4 instructions by my count) * 2 instances is 0.5ms at the nominal 48MHz, it'd be really better to have either a sense that this much is actually necessary (rather than less) or that this is really the "right" solution to the errata.

Reply:
Ugh, yeah. If this is going to keep popping up in places, maybe the better thing would be a dedicated function in the file, i.e.

```rust
// With the comment from 483-488
fn iom_fifo_errata_delay() {
    for _i in 0..3000 {
        cortexm4::support::nop();
}
```

```suggestion
iom_fifo_errata_delay();
```

I suspect this delay will end up other places before this chip is done :/